### ISSUE TITLE
Different results running with and without --format

### ISSUE THREAD

**Issue by FabioAntunes** [Flet/semistandard#47]
So I have this piece of code, which I removed some of the code to make it simpler:
```javascript
(function () {
  'use strict';
  angular.module('dm.popular.selection')
    .controller('PopularSelectionController', PopularSelectionController);

  /* ngInject */
  function PopularSelectionController (PopularSelectionService) {
    var vm = this;

    function getSelections () {
      vm.selection = [];

      PopularSelectionService.getSelections()
        .success(function (data) {
          vm.selectionItems = data.items;
          vm.selectionEngage = data.dateEngage;
          vm.processing = false;
        }).error(function (data) {
        vm.processing = false; //problem here
        vm.dmFormErrors = data; // and here
      });
    }
  }
}());
```

So as you can see, `vm.processing` and `vm.dmFormErrors` should have more two spaces. So I add them:
```javascript
        }).error(function (data) {
          vm.processing = false; //problem here solved
          vm.dmFormErrors = data; // and here also solved
      });
```

Now if I run `semistandard --verbose | snazzy` everything is great and there are no errors, but if I run `semistandard --format | snazzy` it changes my file again to this:
```javascript
        }).error(function (data) {
        vm.processing = false; //problem here again
        vm.dmFormErrors = data; // and here again
      });
```

It removes the two spaces that I added and then throws an error saying that I should add two spaces.

It remembers me when I was young and ask my parents to go out, my Mom would say yes and my Dad would say no and that makes me sad.

Does anyone know if there's something wrong with that specific code or if it's an edge case?

**Comment by Flet** [comment]
The formatter is not fully complete yet unfortunately. 

I'd suggest trying the latest version of `semistandard` as it has a newer version of `semistandard-format` that works a bit better (but its still not quite ready for prime time). Can you give the latest a try?

**Comment by Flet** [comment]
We just published a beta version of the latest `semistandard` (based off the `standard` beta) which also includes the `--fix` flag. `--fix` uses `eslint` to fix quite a few issues and in a more reliable way.

Give it a try? 

```
npm install -g semistandard@beta
semistandard --fix
```

**Comment by Flet** [comment]
We just released `semistandard@9.0.0` which includes the `--fix` flag. I think you'll get a lot better/consistent experience using `--fix` instead of `--format`.

```bash
npm install -g semistandard@latest
semistandard --fix
```

Closing this issue for now. Good luck!
